src/coreComponents/physicsSolvers/fluidFlow/kernelSpecs.json

```diff
@@ -24,125 +24,6 @@
                    "generation call). Duplicate entries are unified internally" ]
   },
 
-  "SinglePhaseHybridFVMKernels": {
```

Comment:
I don't think you want to delete this. perhaps comment it out for when you will come back and change the strategy to autogenerate the dispatch. I would gently recommend that you do that in this PR, but it is your call.

Reply:
Based on our communication, I created the following issue: https://github.com/GEOS-DEV/GEOS/issues/3894